.github/workflows/containers.yaml

```diff
@@ -76,8 +76,7 @@ jobs:
 
       - name: Login to GitHub Packages (ghcr.io)
         if: |
-          ${{ github.event_name != 'pull_request' }} &&
-          ${{ github.ref == 'refs/heads/main' }}
+          ${{ github.event_name != 'pull_request' }}
```

Comment:
I'm not sure I follow how this is better - doesn't this make us attempt to push in more situations?

Reply:
I believe `${{ github.ref == 'refs/heads/main' }}` is "wrong" (aka wasn't the expected condition).
_`'${{ github.ref_name }}' == 'main'` is what is used above (and is working "right")_

Given that https://github.com/jelmer/janitor/pull/971, I was under the impression the idea is to push more?
Same with having the tags - otherwise, a lot of that code wouln't be needed?